modules/core/src/main/scala/org/scalasteward/core/buildtool/sbt/package.scala

```diff
@@ -35,10 +35,22 @@ package object sbt {
       Dependency(sbtGroupId, sbtArtifactId, sbtVersion.value)
     }
 
-  val scalaStewardScalafixSbt: FileData =
+  val sbtScalaFixDependency: Scope[Dependency] =
+    Scope(
+      Dependency(
+        GroupId("ch.epfl.scala"),
+        ArtifactId("sbt-scalafix"),
+        "",
+        Some(SbtVersion("1.0")),
+        Some(ScalaVersion("2.12"))
+      ),
+      List(Resolver.mavenCentral)
```

Comment:
This should be `Config#defaultResolver` which is `Resolver.mavenCentral` by default.